src/ArborX_DistributedSearchTree.hpp

```diff
@@ -121,8 +121,9 @@ DistributedSearchTree<DeviceType>::DistributedSearchTree(
   Kokkos::View<Box *, DeviceType> boxes(
       Kokkos::ViewAllocateWithoutInitializing("rank_bounding_boxes"),
       comm_size);
-  // FIXME when we move to MPI with CUDA-aware support, we will not need to
-  // copy from the device to the host
+  // We could avoid copying between device host and device with CUDA-aware MPI,
+  // but this requires the object to be initialized on the device with host data
+  // (_bottom_tree.bounds()). Currently, it doesn't seem worthwhile trying this.
```

Comment:
Try with device data.  Here is how you can get it.
```C++
using TreeAccess = Details::TreeVisualization<DeviceType>::TreeAccess;
TreeAccess::getBoundingVolume(TreeAccess::getRoot(_bottom_tree), _bottom_tree);
```
PS: I am not advocating that is good design but at least it will tell us if this is worth pursuing at all.

Reply:
This will be explored in a separate PR.

Reply:
See #174.